woocommerce/woocommerce-ios
# Variations: Remove unnecessary no-price warning banner

fixes #4451

# Why

During testing, it was reported that the "no price warning" banner in the variation list screen was redundant because we also say that a variation does not have a price within each cell. 

This PR removes that banner.

# How 

Just 🔪 it!

# Screenshots

Before | After 
--- | ---
<img width="375" alt="before" src="https://user-images.githubusercontent.com/562080/124322456-1830a680-db45-11eb-9491-7d95be771285.png"> | <img width="384" alt="after" src="https://user-images.githubusercontent.com/562080/124322459-1961d380-db45-11eb-8dee-b8b0809924c2.png">


# Testing Steps

- Create a new variable product

- Add a variation

- Navigate to the variations list screen and confirm that the warning banner is not visible.

Update release notes:
